Flow.Launcher.Test/FuzzyMatcherTest.cs

```diff
@@ -142,20 +144,20 @@ public void GivenQueryString_WhenAppliedPrecisionFiltering_ThenShouldReturnGreat
             var rawScore = matcher.FuzzyMatch(queryString, compareString).RawScore;
 
             // Should
-            Assert.AreEqual(expectedScore, rawScore, 
+            Assert.AreEqual(expectedScore, rawScore,
                 $"Expected score for compare string '{compareString}': {expectedScore}, Actual: {rawScore}");
         }
 
         [TestCase("goo", "Google Chrome", SearchPrecisionScore.Regular, true)]
         [TestCase("chr", "Google Chrome", SearchPrecisionScore.Low, true)]
         [TestCase("chr", "Chrome", SearchPrecisionScore.Regular, true)]
-        [TestCase("chr", "Help cure hope raise on mind entity Chrome", SearchPrecisionScore.Regular, false)]
         [TestCase("chr", "Help cure hope raise on mind entity Chrome", SearchPrecisionScore.Low, true)]
         [TestCase("chr", "Candy Crush Saga from King", SearchPrecisionScore.Regular, false)]
         [TestCase("chr", "Candy Crush Saga from King", SearchPrecisionScore.None, true)]
-        [TestCase("ccs", "Candy Crush Saga from King", SearchPrecisionScore.Low, true)]
-        [TestCase("cand", "Candy Crush Saga from King",SearchPrecisionScore.Regular, true)]
-        [TestCase("cand", "Help cure hope raise on mind entity Chrome", SearchPrecisionScore.Regular, false)]
+        [TestCase("ccs", "Candy Crush Saga from King", SearchPrecisionScore.Regular, true)]
```

Comment:
why has this precision changed to regular?

Reply:
Because the acronym search will allow that to be matched with SearchPrecisionScore.Regular